### ISSUE TITLE
@perplexity can't be authorized for individual account

### ISSUE THREAD

**Issue by jiangyan** [microsoft/vscode-copilot-release#2093]
Type: <b>Bug</b>

A page with 'This GitHub App must be configured with a callback URL' poped up, and nothing happened

Extension version: 0.22.2024102902
VS Code version: Code - Insiders 1.95.0-insider (912bb683695358a54ae0c670461738984cbb5b95, 2024-10-28T20:14:20.189Z)
OS version: Windows_NT x64 10.0.27729
Modes:

**Comment by hasokeric** [comment]
Same result in vscode-insiders

**Comment by JacobD2001** [comment]
Same for me.

**Comment by aleivaar94** [comment]
same for me

**Comment by rajeshkanaka** [comment]
Yes as per documentation no extra setup needed , but still getting error as "This GitHub App must be configured with a callback URL" on the first message. 
 ![Image](https://github.com/user-attachments/assets/5aa3f994-db53-408c-a2f9-434e69f3cbe4)

**Comment by honofung1** [comment]
still have the same issue

**Comment by kawnnor** [comment]
Same for me.

**Comment by belaltaher8** [comment]
This issue is resolved. Please retry authenticating the `@perplexityai` extension and it will work now.

**Comment by rajeshkanaka** [comment]
looks like the auth issue resolved but the new issue regarding response born.
![Image](https://github.com/user-attachments/assets/a5713c2a-f0f6-43e4-9da7-455a1432ce23)

**Comment by jiangyan** [comment]
The issue got fixed, thanks!

![Image](https://github.com/user-attachments/assets/24ec6762-8ac2-466a-b57a-0249eb5f8101)

**Comment by rajeshkanaka** [comment]
Yes, finally working after reinstalling it through Github portal and giving access again.
![Image](https://github.com/user-attachments/assets/c2e769c5-4586-4005-8f30-99bcda16fdc4)

**Comment by khantseithu** [comment]
> Yes, finally working after reinstalling it through Github portal and giving access again. ![Image](https://github.com/user-attachments/assets/c2e769c5-4586-4005-8f30-99bcda16fdc4)

I'm having the exact issue, how do I resolve this? I've tried uninstall perplexity and reinstall it again
